### DISCUSSION TITLE
Hábitos e Rotinas

### DISCUSSION THREAD

**Discussion by mariojvr** [frontendbr/forum#994] [Discussão]
Sou DBA Oracle/Sybase e de 1 ano comecei a estudar o universo front-end. Por conta do trabalho, e tentando conciliar a vida, tenho desenvolvido rotinas de estudo, trabalho, leitura, etc.
E gostaria de saber de vocês. Estudam quantas horas por dia? De manhã ou a noite? Prática de exercícios físicos? Enfim, compartilhar um pouco das suas rotinas e hábitos do dia a dia e como conseguem conciliar tudo.

**Comment by mahenrique94** [comment]
Cara eu tento estudar no mínimo uma hora por dia (é pouco, mas melhor um pouco todo dia do que muito apenas um dia), tento estudar no almoço ou antes de dormir, sempre que tenho um tempo livre.

Depois dos estudos gosto de fazer projetos práticos, penso em algo legal e desenvolvo utilizando os conhecimentos teóricos adquiridos com o estudo.

Depois do projeto, gosto de publicar posts no meu blog sobre o assunto.

http://blog.alura.com.br/planejando-meus-estudos/
http://blog.alura.com.br/o-dia-a-dia-de-um-profissional-que-aprende-o-calendario-de-estudo/

Ótimos artigos para se ler.
